ArduSub/radio.cpp

```diff
@@ -0,0 +1,152 @@
+// -*- tab-width: 4; Mode: C++; c-basic-offset: 4; indent-tabs-mode: nil -*-
+
+#include "Sub.h"
+
+
+// Function that will read the radio data, limit servos and trigger a failsafe
+// ----------------------------------------------------------------------------
+
+void Sub::default_dead_zones()
+{
+    channel_roll->set_default_dead_zone(30);
+    channel_pitch->set_default_dead_zone(30);
+    channel_throttle->set_default_dead_zone(30);
+    channel_yaw->set_default_dead_zone(40);
+    channel_forward->set_default_dead_zone(30);
+    channel_lateral->set_default_dead_zone(30);
+}
+
+void Sub::init_rc_in()
+{
+	channel_pitch    = RC_Channels::rc_channel(0);
+    channel_roll     = RC_Channels::rc_channel(1);
+    channel_throttle = RC_Channels::rc_channel(2);
+    channel_yaw      = RC_Channels::rc_channel(3);
+    channel_forward  = RC_Channels::rc_channel(5);
+    channel_lateral  = RC_Channels::rc_channel(6);
```

Comment:
We had issues with QGC calibration, our suggested controls in documentation, and people generally not being familiar with how everything works, so we hard-coded it in order to reduce the number of troubleshooting conversations we were dealing with. I've made a PR concerning joystick calibration in QGC, and we probably will switch back to RC_Mapper at some point.